docs/source/conf.py

```diff
@@ -183,3 +184,11 @@
 
 # If false, no module index is generated.
 # latex_use_modindex = True
+
+# -- Options for intersphinx extension ---------------------------------------
+
+intersphinx_mapping = {
+    "pyface": ("https://docs.enthought.com/pyface", None),
+    "traits": ("https://docs.enthought.com/traits", None),
+    "traitsui": ("https://docs.enthought.com/traitsui", None),
```

Comment:
Is it also worth adding core Python?

Reply:
possibly. not sure exactly how but possibly.

Reply:
It just saves an extra PR later when you realise you want to refer to something in the std. lib. :-)